chance.js

```diff
@@ -7245,7 +7245,9 @@ options,
+          // list of flying animals comes from https://7esl.com/birds-vocabulary-english/
+          "flyers": ["Crested Auklet","Great Auk","Japanese Murrelet","Little Auk","Razorbill","Brewer's Blackbird","Cuban Blackbird","Eurasian Blackbird","Red-Winged Blackbird","Tawny-Shouldered Blackbird","Black-Capped Chickadee","Boreal Chickadee","Carolina Chickadee","Chestnut-Backed Chickadee","Gray-Headed Chickadee","American Crow","Cape Crow","Carrion Crow","Collared Crow","Eastern Jungle Crow","African Collard Dove","Carribean Dove","Common Ground Dove","Eurasian Collared Dove","European Turtle Dove","African Black Duck","American Black Duck","Australian Wood Duck","Black-Headed Duck","Comb Duck","African Hobby","American Kestrel","Amur Falcon","Aplomado Falcon","Australian Kestrel","Brambling","Bullfinch","Chaffinch","Common Redpoll","Common Rosefinch","North American Flycatcher","Eurasian Flycatcher","Mexican Flycatcher","Grey Partridge","Bobwhite Quail","Ringneck Pheasant","Ruffed Grouse","Sharptail Grouse","African Goose","Barnacle Goose","Canada Goose","Crested Goose","Emperor Goose","Andean gull","Common Gull","Gray Gull","Ivory Gull","Laughing Gull","Broad-Winged Hawk","Cooper's Hawk","Ferruginous Hawk","Gray Hawk","Harris's Hawk","American Bittern","Cattle Egret","Glossy Ibis","Great Blue Heron","Great Egret","Allen's Hummingbird","Anna's Hummingbird","Bee Hummingbird (This is the smallest bird!)","Black-Chinned Hummingbird","Broad-Tailed Hummingbird","Amazon Kingfisher","Azure Kingfisher","Belted Kingfisher","Collard Kingfisher","Common Kingfisher","Brown-Headed Nuthatch","Eurasian Nuthatch","Pygmy Nuthatch","Masai Ostrich","North African Ostrich","Southern Ostrich","Barn Owl","Eastern Screech Owl","Great Horned Owl","African Grey","Amazon Parrot","Macaw","Adelie Penguin","Chinstrap Penguin","Emperor Penguin","Berne Lark","Domestic Pigeon","Homing Pigeon","Australian Raven","Common Raven","Forest Rave","Albatross","Booby","Cormorant","American Golden Plover","Baird's Sandpiper"]
```

Comment:
For consistency:
```Andean gull``` should be ```Andean Gull```

Also, I'm not sure if we should be providing facts in the list:
```Bee Hummingbird (This is the smallest bird!)``` should be ```Bee Hummingbird``` 🤷

Do ostriches fly? 🤔